webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue

```diff
@@ -291,12 +374,25 @@ export default {
     isMobile() {
       return this.$vuetify.breakpoint.name === 'xs' || this.$vuetify.breakpoint.name === 'sm';
     },
+    isRedactorSpace() {
+      return this.news.id !== '' && this.currentSpace && this.currentSpace.redactorsCount > 0 && this.currentSpace.managersCount > 0;
```

Comment:
```suggestion
      return this.currentSpace && this.currentSpace.redactorsCount > 0;
```

Reply:
this.news.id !== '' it's important to display the selector only if we tab at least text.